lms/djangoapps/teams/tests/test_views.py

```diff
@@ -186,7 +202,7 @@ def make_call(self, url, expected_status=200, method='get', data=None, content_t
 
         If a user is specified in kwargs, that user is first logged in.
         """
-        user = kwargs.pop('user', 'student_enrolled')
+        user = kwargs.pop('user', 'student_enrolled_not_on_team')
```

Comment:
@explorerleslie what do you want this to look like? Do you want just an error that displays if you try to create a second team saying that you are already a member of the team, or do you want the Create A New Team button disabled/missing (with some sort of messaging)? We will need to also handle this in the UI for adding yourself as a member of a team.